setup.cfg

```diff
@@ -38,14 +37,15 @@ ignore =
     H306,
     # W503:  line break before binary operator (this is no longer PEP8 compliant)
     W503,
+    # E203: whitespace before ':' (this is not PEP8 compliant)
+    E203,
```

Comment:
Why ignore E203?

Why ignore E203 in a "Move to src dir layout" commit?

Reply:
We ignore `E203` as it's not compatible with `black` or `Pep8`. Granted that probably should've been in the "merge mbed-build" commit (as that's the repo it came from), or a separate commit.

Reply:
OK, it looked like this came out of the blue. Fine with this going into mbed-build merge commit.